### DISCUSSION TITLE
Dynamic parameters binding of string argument

### DISCUSSION THREAD

**Discussion by michal-bartak** [dbeaver/dbeaver#18375] [General]
Hello
Let's consider following query
```sql
SELECT *
FROM mytable
WHERE mytxtcolumn = 'abc';
```

How to use dynamic parameters binding?
The example below doesn't work
```sql
SELECT *
FROM mytable
WHERE mytxtcolumn = ':name';
```

I can use it without single-quotes, but then I need to enter them in argument popup
```sql
SELECT *
FROM mytable
WHERE mytxtcolumn = :name;
```
![image](https://user-images.githubusercontent.com/7868445/204511191-47edd689-7d37-4162-8357-b13cfeab7d7d.png)

**Comment by michal-bartak** [comment]
Anybody can help?
Should we convert it to the issue?

**Reply by E1izabeth** [reply]
Hi, we have already fixed it in scope of this issue #18885
The fix will be available in the future release.

**Reply by michal-bartak** [reply]
Ahh nice. Thank you.
